cmd/humanlog/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestApplyConfigFromConfigFile_when_one_of_skip_or_keep_is_given(t *testing.T) {
+
+	wd, _ := os.Getwd()
+	dirs := strings.Split(wd, "/")
+	root := strings.Join(dirs[:len(dirs)-2], "/")
+	configFilePath := root + "/test/cases/00065-apply-config/config.json"
```

Comment:
can you switch this to use https://pkg.go.dev/testing#T.TempDir and write a file there?

you can use this struct: https://github.com/humanlogio/humanlog/blob/master/internal/pkg/config/config.go#L78-L98

maybe something like (pseudo-code):
```go
cfg := config.Config{
   Skip: []string{"foo", "bar"},
}

dir := t.TempDir()
f, err := os.TempFile(dir, "00065-apply-config.json)
require.NoError(t, err)

err = json.NewEncoder(f).Encode(cfg)
require.NoError(t, err)

err = f.Close()
require.NoError(t, err)

args := []string{"program-path", "--config", f.Name()}
...
```